x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Observable } from 'rxjs';
+import moment from 'moment';
+import { OpsMetrics } from 'kibana/server';
+import { KIBANA_STATS_TYPE_MONITORING } from '../../../common/constants';
+
+/*
+ * Initialize a collector for Kibana Ops Stats
+ */
+export function getOpsStatsCollector(usageCollection: any, metrics$: Observable<OpsMetrics>) {
```

Comment:
I think you can use: `UsageCollectionSetup` type for `usageCollection`

Reply:
Nice catch!